src/lib.rs

```diff
@@ -159,6 +159,9 @@
 
 #![doc(html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
        html_favicon_url = "https://www.rust-lang.org/favicon.ico")]
+
+#![doc(html_root_url = "https://docs.rs/semver/0.9.0")]
```

Comment:
We could simplify this a little by adding the `html_root_url` to the same `doc` attribute:

```rust
#![doc(html_root_url = "https://docs.rs/semver/0.9.0",
       html_logo_url = "https://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
       html_favicon_url = "https://www.rust-lang.org/favicon.ico")]
```